### PR TITLE
Added dutch language to the time elements and modified the time eleme…

### PR DESCRIPTION
**Pull request by lenmen**
now the times are in dutch notation ^.^cheers

### CODE REVIEW

File: resources/languages/nl/corpus/time.clj [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+  "momenteel"
+  (datetime 2017 1 27)
+
+  "gisteren"
+  (datetime 2017 1 26)
+
+  "morgen"
+  (datetime 2017 1 28)
+
+  "morgen avond"
```
Comment (robinvdvleuten) [inline resources/languages/nl/corpus/time.clj]: I would like to see here "morgenavond" as well, that's the more common type of writing.

File: resources/languages/nl/corpus/time.clj [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+  "eergisteren"
+  (datetime 2017 1 25)
+
+  "overmorgen"
+  (datetime 2017 1 29)
+
+  "maandag ochtend"
+  (datetime-interval [2017 1 30 04] [2017 1 30 12])
+
+  "gisterenochtend"
```
Comment (robinvdvleuten) [inline resources/languages/nl/corpus/time.clj]: This should be "gisteravond" in dutch

File: resources/languages/nl/corpus/time.clj [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+  (datetime-interval [2017 1 26 4] [2017 1 26 12])
+
+  "gisterenavond"
```
Comment (robinvdvleuten) [inline resources/languages/nl/corpus/time.clj]: Same here, "gisteravond"

File: resources/languages/nl/corpus/time.clj [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+
+  ; Part of day (morning, afternoon...)
+
+  "vanavond"
+  (datetime-interval [2017 1 27 18] [2017 1 28 00])
+
+  "dit weekend"
+  (datetime-interval [2017 1 27 18] [2017 1 30 00])
+
+  "morgen middageten"
```
Comment (robinvdvleuten) [inline resources/languages/nl/corpus/time.clj]: "morgenmiddag eten"

File: resources/languages/nl/corpus/time.clj [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+  "zondag"
+  (datetime 2017 1 29 :day-of-week 7)
+
+  "dinsdag"
+  (datetime 2017 1 31 :day-of-week 2)
+
+  ;; Hours
+
+  "om drie uur"
+  "om drie uur"
```
Comment (robinvdvleuten) [inline resources/languages/nl/corpus/time.clj]: Double entry

File: resources/languages/nl/rules/time.clj [hunk 6]
```diff
@@ -0,0 +1,467 @@
...
+  "named-day"
+  #"(?i)maandag|ma"
+  (day-of-week 1)
+
+  "named-day"
+  #"(?i)dinsdag|di"
+  (day-of-week 2)
+
+  "named-day"
+  #"(?i)woensdag|wo"
```
Comment (robinvdvleuten) [inline resources/languages/nl/rules/time.clj]: "woe" should be valid as well.

File: resources/languages/nl/rules/time.clj [hunk 7]
```diff
@@ -0,0 +1,467 @@
...
+  (day-of-week 3)
+
+  "named-day"
+  #"(?i)donderdag|do"
+  (day-of-week 4)
+
+  "named-day"
+  #"(?i)vrijdag|vr"
```
Comment (robinvdvleuten) [inline resources/languages/nl/rules/time.clj]: "vrij" should be valid as well.

File: resources/languages/nl/rules/time.clj [hunk 8]
```diff
@@ -0,0 +1,467 @@
...
+  (day-of-week 5)
+
+  "named-day"
+  #"(?i)zaterdag|za"
```
Comment (robinvdvleuten) [inline resources/languages/nl/rules/time.clj]: "zat" should be valid as well.

**Comment by robinvdvleuten** [comment]
Before anyone wonders why I did the review. @lenmen is a colleague of mine and asked to do the review as well 😄

**Comment by lenmen** [comment]
fixed comments. @robinvdvleuten thanks for the review^^

**Comment by robinvdvleuten** [comment]
@lenmen thanks for the evening effort 👍

**Comment by robinvdvleuten** [comment]
Any updates on this PR?

**Comment by Woutje99** [comment]
Hi, can someone say when this is adding to the site?

**Comment by joachimdb** [comment]
+1

**Comment by ar7hur** [comment]
So sorry guys for the delay... just merged. Thanks!

**Comment by Woutje99** [comment]
When will it be available on wit.ai?

**Comment by ar7hur** [comment]
@Woutje99 per https://wit.ai/blog/2017/05/01/new-duckling wit.ai moved to a new Haskell-based Duckling (in particular to handle a fast-growing load). The new Haskell-based Duckling is at https://github.com/facebookincubator/duckling it's still open source and you can contribute rules (the format is slightly different, but the philosophy is the same).

**Comment by Woutje99** [comment]
@ar7hur Yeah I read that. But does it mean that this translation not will be used on Wit.ai?

**Comment by ar7hur** [comment]
@Woutje99 yes, unless of course somebody pulls it into the Haskell version

**Pull request merged**